### ISSUE TITLE
36684702 invalid birth date (Julie)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#739]
Issue reported by **JHarold** at 2016-02-29 14:11:42 UTC
Time: 2016-02-29T14:07:36+00:00
Session ID: 1deaaf553e0a46b0db4519a5e6fc218c
Problem Page URL: [/freereg1_csv_files/56d44c70f493fde7c6000001/error](/freereg1_csv_files/56d44c70f493fde7c6000001/error)
Previous Page URL: [http://freereg2.freereg.org.uk/freereg1_csv_files](http://freereg2.freereg.org.uk/freereg1_csv_files)
Reported Issue:
Can you tell me what is wrong with the birth dates classed as errors here ?  Is there some hidden text ?

I converting donated files in word format, via excel into our format.  I have done a number recently and a lot of them have had this problem with the birth dates (not all of them).

They look fine in csv file, and fine online.  To correct it I have been deleting the date and retyping it online.  It is getting quite tedious and would be better to screen for the error before uploading - but I don''t know what to look for.

ef userid JHarold   batch ESSPAGBA and ESSNSHBA

Julie (Essex coordinator)    

![Screenshot](http://www.freereg.org.uk)

**Comment by edickens** [comment]
I can't see anything wrong.  E

**Comment by Sherlock21** [comment]
Julie   

```
Try the conversion on you Mac.
```

I had another issue where something that had bee created in non windows spreadsheet went to rats in XLSX and from there I was trying to read it in Mac - it generated extra blank rows and other rubbish. A google search advised me it was a know issue trying to convert into Windows  

I dont remember the exact details - but its a clue.

otherwise email me on of you files if you wish?

Where is telling you there are dates issue?

ericB

**Comment by Captainkirkdawson** [comment]
We need an explicit example

**Comment by edickens** [comment]
There was a screenshot in original message but it doesn't work now.  Julie said userid JHarold batch ESSPAGBA and ESSNSHBA
If not enough information we need to ask Julie.

**Comment by Captainkirkdawson** [comment]
Leave it with you

**Comment by edickens** [comment]
I am sure that I answered it earlier, but where has my comment gone?
Julie said:-
just to let you know I have figured out the problem,  The original was
a word document which contained some non breaking spaces which were
then being converted to disallowed characters which the freereg site
was objecting to.  I have found a way to convert them before shifting
to CSV format.
So unless FR is to be altered to accept the non-breaking spaces, then this can be closed but something should be put in the Help.
